test/parallel/test-runner-run.mjs

```diff
@@ -551,3 +551,9 @@ describe('forceExit', () => {
     });
   });
 });
+
+// exitHandler doesn't run until after the tests / after hooks finish.
+process.on('exit', () => {
+  assert.strictEqual(process.listeners('uncaughtException').length, 0);
+  assert.strictEqual(process.listeners('unhandledRejection').length, 0);
+});
```

Comment:
I think you need to wrap this is in a `common.mustCall`.

Reply:
I don't believe `common.mustCall()` works here. I tried using it but the mustCall check errors because it runs it's check on `process.exit()` as well.

I found this warning too: https://github.com/nodejs/node/blob/cd39578e53d6502de03f15620d15835b86954ac8/test/common/index.js#L462

Reply:
Ok, good catch!